RussianFilter/sections/specific.txt

```diff
@@ -54,6 +54,8 @@ focus.ua##.bnr
 ||acc.cv.ua/images/bimg/
 prikol.ru##.post.category-reklama
 libraryno.ru##div[class^="br_goo_"]
+prophotos.ru##div[class^="branding"]
```

Comment:
I guess this rule is superfluous, because next one already blocking this element

Reply:
agree. But the first rule is also working without the second